src/sentry/api/endpoints/group_notes_details.py

```diff
@@ -26,8 +27,24 @@ def delete(self, request: Request, group, note_id) -> Response:
         except Activity.DoesNotExist:
             raise ResourceDoesNotExist
 
+        webhook_data = {
+            "comment_id": note.id,
+            "timestamp": note.datetime,
+            "comment": note.data.get("text"),
+            "project_slug": note.project.slug,
+        }
+
         note.delete()
 
+        if Activity.objects.filter(id=webhook_data["comment_id"]).count() == 0:
```

Comment:
You don't need to make another query to verify deletion. `Model.delete()` returns the number of objects deleted (see: https://docs.djangoproject.com/en/4.0/topics/db/queries/#deleting-objects).